README.md

```diff
@@ -57,13 +100,15 @@ Run `ng g component my-component --project=my-app` to generate a new component.
 
 Run `ng build my-app` to build the project. The build artifacts will be stored in the `dist/` directory. Use the `--prod` flag for a production build.
 
-## Running unit tests
+## Tests
+
+### Running unit tests
```

Comment:
```suggestion
- Running unit tests
```
Propose that you have a single title: Testing
and then you have bulleted items or in a table: the different commands
Keep it simple.